marlowe/shell.nix

```
@@ -0,0 +1,4 @@
+let
+  local = import ../. {};
+in
+local.localLib.withDevTools local.localPackages.marlowe.env
```

I changed this to be even simpler recently.
```suggestion
(import ../. {}).shellTemplate "marlowe"
```